app/src/androidTest/java/com/example/android/architecture/blueprints/todoapp/data/source/local/TasksDaoTest.kt

```diff
@@ -38,7 +36,12 @@ import org.junit.runner.RunWith
 @SmallTest
 class TasksDaoTest {
 
-    private lateinit var database: ToDoDatabase
+    // using an in-memory database because the information stored here disappears when the
+    // process is killed
+    private val database = Room.inMemoryDatabaseBuilder(
```

Comment:
We probs still want to do this in `@Before` and `@After` so results of prior tests in the same test suite don't affect new ones

Reply:
How would tests affect previous tests if we're calling `clearAllTables` @Before each test?